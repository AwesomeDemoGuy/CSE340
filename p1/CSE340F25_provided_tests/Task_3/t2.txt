TASKS
1 3
POLY
F1 = x^2 + 1;
F2(x,z,y) = y x^2 x^3 + y x z + z^2 y + 1;
EXECUTE
INPUT X;
INPUT Y;
W = F4(X,Y, F2(X,Y,X));
OUTPUT Y;
INPUTS
5 2 3 18 19 22 33 12 11 16