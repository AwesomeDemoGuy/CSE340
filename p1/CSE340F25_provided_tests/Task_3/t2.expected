F1 = x^2 + 1
F2(x,z,y) = x^5 y + x z y + z^2 y + 1